hc/views/views_receptionist.py

```diff
@@ -38,3 +55,11 @@ def SearchAppointmentView(request, username):
             form = ViewAppointmentForm(initial={'patient': username})
             appn = None
             return render(request, 'receptionist/view_appointment.html', {'form': form, 'appointment': appn})
+
+
+def Recep_Appointments(request):
+    data = val()
+    field = data['doctor']
```

Comment:
why are you getting the doctor via the global function! this is a really really bad way to code!
Pass the doctor id through the URL!
```/receptionist/appointments/<doctor_username>```